reactioncommerce/reaction
# Update email for inviting a user to create a shop

Resolves #2804. 
_ Updates email copy for new shop owner and uses the primary shop details (name, address etc) when sending email to new owner (instead of the newly created shop details as before).
_ Also updates the email copy sent to a user when added to a group (see ticket for the copy #2804)

**How To Test:**
- Invite a new owner/merchant under Shop > Marketplace in dashboard
- Confirm updated copy in the email sent to the merchant
- Invite a new user to a group in Accounts > Add Admin User in dashboard in Primary Shop
- Confirm updated copy in the email sent to the account
- Invite a new user to a group like above but in a merchant shop
- Confirm that email copy sent is also updated (and it reflects the right shop name)
